### ISSUE TITLE
Determine hyper parameter by cross-validation

### ISSUE THREAD

**Issue by orion0616** [orion46/deep-learning#1]
Like chap3, I need to determine good hyper parameter to avoid overfitting in chap4. Using cross-validation, I have to choose appropriate hyper parameter(the number of 1st layer, learning rate and the variance of initial weights).

**Comment by orion0616** [comment]
This issue is low priority. close
